generate/metricbeat/metricset/README.asciidoc

```diff
@@ -20,32 +21,36 @@ The first step is to clone the beats repository:
 git clone https://github.com/elastic/beats $GOPATH/src/github.com/elastic/beats
 ----
 
-After having cloned the beat repository into the `GOPATH`, change to the directory where you want to create the beat.
- This is normally under `$GOPATH/src/github.com/{your-github-com}`. Inside this directory run the command:
+After cloning the beats repository into the `GOPATH`, change to the directory where you want to create the beat.
+This directory is normally located under `$GOPATH/src/github.com/{your-github-com}`. Inside this directory, run the command to create the beat.
 
 
 [float]
-=== Step 2 - Create Beat
+=== Step 2 - Create the Beat
+
+Run the command:
 
 [source,bash]
 ----
 cookiecutter $GOPATH/src/github.com/elastic/beats/generate/metricbeat/metricset
 ----
 
-This will ask you some question about the beat name and path. Complete it accordingly.
+When prompted, enter the Beat name and path.
 
 
 [float]
-=== Step 3 - Init and Create MetricSet
+=== Step 3 - Init and create the metricset
+
+After creating the Beat, change to the directory for your Beat and run:
```

Comment:
Not sure this is correct. Wouldn't it be `change the directory to your Beat`? Making that same assumption as above with Github, we could also say:

Change the directory to `$GOPATH/src/github.com/{your-github-com}/{beat}`. Does that make sense?